tree.go

```diff
@@ -468,6 +463,9 @@ func (n *InternalNode) CreatePath(path []byte, stemInfo stemInfo, comms []*Point
 	return child.CreatePath(path[1:], stemInfo, comms, values)
 }
 
+// GetStem returns the all NodeWidth values of the stem.
+// The returned slice is internal to the tree, so it *must* be considered readonly
+// for callers.
 func (n *InternalNode) GetStem(stem []byte, resolver NodeResolverFn) ([][]byte, error) {
```

Comment:
Interestingly, this `GetStem(...)` was 100% uncovered.

Then I realized that `Get(key)` was having a 99% similar implementation than this method, but that didn't make much sense. `Get(..,...)` a more concrete action than `GetStem(...)`. To avoid duplication, I made `Get(...)` call `GetStem(...)` as you'll see later, and avoid repeated code.

See the next comment.